config/application.yml.default

```diff
@@ -454,7 +454,7 @@ production:
   doc_auth_vendor_randomize_alternate_vendor: ''
   domain_name: login.gov
   email_registrations_per_ip_track_only_mode: true
-  enable_test_routes: false
+  enable_test_routes: true
```

Comment:
Would not expect this change.

```suggestion
  enable_test_routes: false
```

Reply:
Yup - was trying to get the review app test routes to work - didn't realize there was a PR for this branch! Cleaned up now.